p2p/src/net/mock/transport/tcp.rs

```diff
@@ -40,58 +42,75 @@ use crate::{
     P2pError, Result,
 };
 
+use self::encryption::{Encryption, NoiseEncryption};
+
 #[derive(Debug)]
-pub struct TcpMockTransport {}
+pub struct TcpMockTransportBase<E: Encryption>(std::marker::PhantomData<E>);
+
+// By default the transport uses Noise protocol encryption
```

Comment:
No, it shouldn't. There's no default. The instantation of TcpMockStream should establish whether we want encryption or not.

Reply:
Default removed. Will that work?